### PR TITLE
improved install langs

### PR DESCRIPTION
**Pull request by Kamahl19**

### CODE REVIEW

File: root/socialnet/install_sn.php [hunk 1]
```diff
@@ -51,7 +51,6 @@
  */
 
 $user->add_lang(array('ucp', 'mods/socialnet'));
-$language_file = 'mods/socialnet_acp';
```
Comment (senky) [inline root/socialnet/install_sn.php:54]: why did you remove language file?

Comment (Kamahl19) [inline root/socialnet/install_sn.php:54]: because it is not used anywhere

Comment (senky) [inline root/socialnet/install_sn.php:54]: Alright

**Comment by senky** [comment]
MA

File: root/socialnet/install_sn.php (hunk 1 above)
Comment (Culprit-CZ) [inline root/socialnet/install_sn.php:54]: Disagree, language file is used for lang to ADMIN EVENT LOG

**Comment by Culprit-CZ** [comment]
Kde je description?????

Jinak bez souboru to bude v ACP -> System -> Admin log davat spatne infromace

File: root/socialnet/install_sn.php (hunk 1 above)
Comment (Kamahl19) [inline root/socialnet/install_sn.php:54]: ah, you are right

File: root/socialnet/install_sn.php [hunk 2]
```diff
@@ -30,7 +30,7 @@
 }
 
 // The name of the mod to be displayed during installation.
-$mod_name = 'Social Network';
+$mod_name = 'phpBB Social Network';
```
Comment (senky) [inline root/socialnet/install_sn.php:33]: I do not like it does not use language file. Or it is not able to be used here?

File: root/socialnet/update_sn_0.7.0.php [hunk 3]
```diff
@@ -30,7 +30,7 @@
 }
 
 // The name of the mod to be displayed during installation.
-$mod_name = 'Social Network - update';
+$mod_name = 'phpBB Social Network update';
```
Comment (senky) [inline root/socialnet/update_sn_0.7.0.php:33]: the same - I do not like it does not use language file (if it can)

Comment (Kamahl19) [inline root/socialnet/update_sn_0.7.0.php:33]: Yes we can use lang files in there, but we have never used them for umil files. However you are right, we should use it.

File: root/socialnet/install_sn.php (hunk 1 above)
Comment (Culprit-CZ) [inline root/socialnet/install_sn.php:54]: Better to use instead 2 lines 53 and 54 ony 1

``` php
$language_file = array('ucp', 'mods/socialnet', 'mods/socialnet_acp');
```

File: root/socialnet/update_sn_0.7.0.php (hunk 3 above)
Comment (senky) [inline root/socialnet/update_sn_0.7.0.php:33]: So do not forget about assigning this variable **after** language file is loaded ;)

**Comment by Culprit-CZ** [comment]
Tys to jeste neudelal?

**Comment by senky** [comment]
Replaced with #31
